manim/animation/updaters/mobject_update_utils.py

```diff
@@ -56,8 +56,32 @@ def updater(mob):
 
 
 def always_redraw(func):
```

Comment:
while we are at it, we could probably also add a type hint for `func`, it should be something like `Callable[[], Mobject]` (with `Callable` from `collections.abc`) I guess.